docs/ref/models/fields.txt

```diff
@@ -1157,6 +1157,35 @@ you can use the name of the model, rather than the model object itself::
         # ...
         pass
 
+Note that relationships defined this way on
```

Comment:
I try to remove phrases like "Note that" from my own writing as they're usually unnecessary.